lms/templates/video.html

```diff
@@ -154,19 +154,22 @@ <h3 class="hidden">${_('No playable video sources found.')}</h3>
 </div>
 % if cdn_eval:
 <script>
-  function sendPerformanceBeacon(id, expgroup, value) {
-    var data = {event: "canplaythrough", id: id, expgroup: expgroup, value: value, page: "html5vid"};
+  function sendPerformanceBeacon(id, expgroup, value, event_name) {
+    var data = {event: event_name, id: id, expgroup: expgroup, value: value, page: "html5vid"};
     $.ajax({method: "POST", url: "/performance", data: data});
   }
-  var cdnStartTime, beaconSent = false;
+  var cdnStartTime;
+  var salt = Math.floor((1 + Math.random()) * 0x100000).toString(36);
   function initializeCDNExperiment() {
+    sendPerformanceBeacon("${id}", ${cdn_exp_group}, "", "load");
     cdnStartTime = Date.now();
-    $("#video_${id}").bind("html5:canplaythrough", null, function() {
-      if (!beaconSent) {
+    $.each(['loadstart', 'abort', 'error', 'stalled', 'loadedmetadata',
+                    'loadeddata', 'canplay', 'canplaythrough', 'seeked'],
```

Comment:
Case is correct -- these are taken from "common/lib/xmodule/xmodule/js/src/video/02_html5_video.js".

seek and seeked are two different events. W3C has a page that demos all the events (http://www.w3.org/2010/05/video/mediaevents.html).